src/components/card/card.tsx

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+
+import { Link } from "gatsby";
+
+import Category from "../category/category.tsx";
+
+import * as cardStyles from "./card.module.scss";
+
+const Card = ({ data }) => (
```

Comment:
Please change the file extension to `jsx` and define the PropTypes for this component

Reply:
Also add a index.js so you avoid importing the component with `/card/card.jsx`

Reply:
Done